fastapi/routing.py

```diff
@@ -645,6 +645,13 @@ def decorator(func: DecoratedCallable) -> DecoratedCallable:
 
         return decorator
 
+    def delete_api_route(self, path: str):
```

Comment:
I would name it `exclude_routes` which should accept an iterable.

Reply:
I also changed the data structure I use. Because if it should accept iterable, it is more efficient to use a dictionary to check if a path exists in self.routes or not.